packages/paypal-js/types/components/buttons.d.ts

```diff
@@ -201,122 +201,155 @@ type OnShippingAddressChangeActions = {
 
 export type DisplayOnlyOptions = "vaultable";
 
+export type PayPalButtonCreateOrder = (
```

Comment:
This is totally my fault for putting this in the ticket, but wanted to throw this out there:
Do you think we should prefix these types with `PayPalButtons___` instead of `PayPalButton___` to match `PayPalButtonsComponentOptions`? 🤔   

Open to either option 😄  But just thought I'd start the conversation.

Reply:
I think `PayPalButton__` is better since we split them into a single button.